.github/workflows/test.yml

```diff
@@ -64,25 +64,43 @@ jobs:
         - { os: ubuntu-24.04-arm, ruby: '2.0' }
         - { os: ubuntu-24.04-arm, ruby: '2.1' }
         - { os: ubuntu-24.04-arm, ruby: '2.2' }
+        # These older Rubies do not exist on Windows arm64
```

Comment:
Actually it's because RubyInstaller doesn't have builds, right? (and of course these older versions might not compile on Windows arm64).
If so, maybe `# RubyInstaller windows-arm64 builds only exist for Ruby 3.4+`

Reply:
From ruby installer’s release note:

> The ARM version required some changes to the side-by-side DLL loading mechanism and it is solved in such a way, that also fixes [a longstanding issue on x64/x86](https://github.com/oneclick/rubyinstaller2/issues/60). Unfortunately these changes are too heavy to be backported to ruby-3.3 and older, so that no ARM package of these older ruby versions will be released.

Reply:
Updated.